packages/js/product-editor/src/blocks/downloads/block.json

```diff
@@ -0,0 +1,26 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "woocommerce/product-downloads-field",
+	"title": "Product downloads",
+	"category": "widgets",
+	"description": "The product downloads.",
+	"keywords": [ "products", "downloads" ],
+	"textdomain": "default",
+	"attributes": {
+		"name": {
```

Comment:
I think that we're not using the [attribute 'name'](https://github.com/woocommerce/woocommerce/pull/40526/files#diff-4d3fbd25af47c29cf34a06bd1ba13f25dba78e1a3f19034c1a67c28c89a3dc58R267), right?

Reply:
Nope but it's required to set `"__experimentalRole": "content"` which will allow to select text under the block.